### ISSUE TITLE
Finding the rule which breaks things

### ISSUE THREAD

**Issue by birdie-github** [Cimbali/CleanLinks#93]
Let's say a certain redirect is broken due to the extension rules.

It would be great to see/find which rule is responsible for the breakage and remove or whitelist it.

Currently it's very hard.

**Comment by Cimbali** [comment]
Agreed. With the UI improvements from #87 it should be somewhat easier, but still hard to parse through a lot of noise when a lot of URLs are cleaned.

I’m not sure how to approach this yet.
